include.js

```diff
@@ -2071,6 +2071,21 @@ let cards = [
     author: 'Gabriel Conde',
     githubLink: 'https://github.com/gabriel-conde'
   },
+  {
+    artName: 'Square', 
+    pageLink: './Art/K-Wiczling/square.html', 
+    imageLink: './Art/K-Wiczling/square.gif', 
+    author: 'K-Wiczling',
+    githubLink: 'https://github.com/K-Wiczling'
+  },
+  {
+    artName: 'Circle', 
+    pageLink: './Art/K-Wiczling/circle.html', 
+    imageLink: './Art/K-Wiczling/circle.gif', 
```

Comment:
Can you please add this file (`circle.gif`) to the contribution? It's missing in this pull request.

Reply:
@K-Wiczling You now added a file called `circle.png`. Thank you 🙏 . But as it's a `.png`, you need to update your `include.js` here on line 2084:
```js
imageLink: './Art/K-Wiczling/circle.png',
```